.rubocop.yml

```diff
@@ -33,3 +35,9 @@ Style/FormatStringToken:
 
 Rails/FilePath:
   EnforcedStyle: arguments
+
+RSpec/HookArgument:
+  EnforcedStyle: each
+
+RSpec/ContextWording:
```

Comment:
Default of `ContextWording` is [(Ref)](https://github.com/rubocop-hq/rubocop-rspec/blob/master/config/default.yml#L65-L71)
```
RSpec/ContextWording:
  Description: Checks that `context` docstring starts with an allowed prefix.
  Enabled: true
  Prefixes:
  - when
  - with
  - without
```

As our convention for context is `context block descriptions should always start with ‘when’ or ‘given’`
cr. Compass

Should we do something like this instead of `Enabled: false`?

```
RSpec/ContextWording:
  Prefixes:
  - when
  - given
```

Reply:
Thanks for the comment, I fixed in https://github.com/nimblehq/rails-templates/pull/246/commits/f8c667c25f1d910968a3c731b4697074165d50c5 @gutakk